graphql-api/src/graphql/types/gene.graphql

```diff
@@ -55,6 +86,7 @@ type Gene {
   mitochondrial_coverage(dataset: DatasetId!): [MitochondrialCoverageBin!] @cost(value: 5)
 
   short_tandem_repeats(dataset: DatasetId!): [ShortTandemRepeat!]! @cost(value: 5)
+  variant_cooccurrence_counts: VariantCooccurrenceCountsPerSeverityAndAf! @cost(value: 5)
```

Comment:
this is something I ought to revisit, because the current version of this imports much less data than my earlier cuts at it (well under half of the original amount), so I suspect that cost might be higher than is appropriate now. I don't know how you'd pick a good cost based on first principles, so my strategy is to see what costs other fields of comparable size have.